dotcom-rendering/Containerfile

```diff
@@ -4,11 +4,11 @@ FROM node:18.17.0-alpine
 
 WORKDIR /opt/app/dotcom-rendering/dotcom-rendering
 
-COPY ./dotcom-rendering/dist /opt/app/dotcom-rendering/dotcom-rendering/dist
+COPY ./dotcom-rendering/target /opt/app/dotcom-rendering/dotcom-rendering
```

Comment:
Was a little hard to grok where this directory layout comes from. Maybe a comment like:
```
# Expects the directory layout created by build-riffraff-bundle
```